tests/kafka/README.md

```diff
@@ -0,0 +1,17 @@
+# Kafka Sink Integration Test
+
+### Start kafka
```

Comment:
I do not (hence the [WIP]). This was something I did in my spare time. I will be moving next month so my spare time is being usurped so I was hoping this would get the ball rolling for kafka.